.github/workflows/ci.yml

```diff
@@ -9,25 +9,6 @@ on:
       - main
 
 jobs:
-  rustfmt:
-    name: RustFmt Check
-    runs-on: ubuntu-latest
-    timeout-minutes: 45
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Install Rust toolchain
-        uses: actions-rust-lang/setup-rust-toolchain@v1
-        with:
-          components: rustfmt
-          rustflags: ""
```

Comment:
Do we lose the ability to see rustfmt's suggestions inlin in PR's code by this change?